plans/init.pp

```diff
@@ -1,7 +1,9 @@
-# A plan that retrieves facts and stores in the inventory for the
-# specified targets.
+# @summary
+#  A plan that retrieves facts and stores in the inventory for the
+#  specified targets.
 #
-# The $targets parameter is a list of targets to retrieve the facts for.
+# @param targets List of targets to retrieve the facts for.
+# @return ResultSet of Task results.
```

Comment:
We don't use the return metadata in Bolt at all. No harm having it there, just a heads up that I doubt anyone will see it unless they read the plan itself (it doesn't look like it gets used in PE Pal either, but maybe it gets used elsewhere?)